lib/beaker-rspec/helpers/serverspec.rb

```diff
@@ -80,7 +80,7 @@ def detect_os
     return Specinfra.configuration.os if Specinfra.configuration.os
     backend = Specinfra.backend
     node = get_working_node
-    if /windows/.match?(node['platform'])
+    if node['platform'].include?('windows')
```

Comment:
This line was also changed in a previous commit. I think if you order this before f91615f4c52e7c2dc9430bcc3b3554eeb907b490 (rubocop: fix Performance/RegexpMatch) the you have fewer changes.

Reply:
I updated the ordering